openai/completions.ts

```diff
@@ -43,26 +54,32 @@ AI: ${i.reply || ""}`
 export async function queryCompletionsChat(
   context: string,
   interactions: ChatInteraction[],
-  options: { apikey: string }
+  options: { apikey: string },
+  isGpt35Turbo = false
```

Comment:
Instead of adding this boolean, we could add the option `model: Model` to the `options` object.

```
type Model = "gpt-3.5-turbo" | "text-davinci-003"
```